src/plugins/intel_npu/tools/protopipe/src/parser/config.cpp

```diff
@@ -134,6 +135,23 @@ static std::string toPriority(const std::string& priority) {
 static ScenarioGraph buildGraph(const std::vector<OpDesc>& op_descs,
                                 const std::vector<std::vector<std::string>>& connections);
 
+static void validateNodeChildren(const YAML::Node& node, std::set<std::string> supportedKeys)
+{
+    auto isScalar2 = node.IsScalar();
+    auto isSequence2 = node.IsSequence();
+    auto isMap2 = node.IsMap();
```

Comment:
what is purpose of these variables?

Reply:
This line is needed for debug. It will be removed when we reach an final agreement.

Reply:
Lines removed